src/PerfView/EtwEventSource.cs

```diff
@@ -210,6 +210,22 @@ public override void ForEach(Func<EventRecord, bool> callback)
                             opcode = TraceEventOpcode.Stop;
                     }
 
+                    if (data.ProviderGuid == systemDataProviderGuid)
+                    {
+                        corelationOptions = CorelationOptions.UseActivityID;
+                        if ((int)data.ID == 1)          // BeginExecute
+                        {
+                            task = (TraceEventTask)0xFFFE;      // unique task but used for both BeginExecute and EndExecute. 
```

Comment:
:bulb: It looks like this constant is a common pattern; seems like a candidate for extraction to a named constant.